locustfile.py

```diff
@@ -384,6 +403,14 @@ def load_schedule_a_problematic(self):
             "schedules/schedule_a/", name="load_schedule_a_problematic", params=params
         )
 
+    # @locust.task
```

Comment:
@qqss88 did you want this to be commented out? I don't think `locust` will run this test if it's not defined as a task.